### PR TITLE
README.md: Using multiple Flymake checkers

### PR DESCRIPTION
**Pull request by hacktivista**
Explains https://github.com/joaotavora/eglot/issues/268 solution

### CODE REVIEW

**Comment by joaotavora** [comment]
This is way too wordy.  Just add an entry after `eglot-server-initialized-hook`, in the same section.  Add a properly formatted link to the issue if you want.

**Comment by hacktivista** [comment]
I think it deserves to be prominent. You are altering an external package's behavior :/

**Comment by joaotavora** [comment]
> You are altering an external package's behavior :/

Eglot leverages a zillion packages, "altering" their behaviour all the time.  You might say it overrides a user's defcustom.  That's true but there's lots of prior art for it, just apropos for "overriding". And that's the basic contract you enter when you type `M-x eglot`.  It says it starts "managing your buffers", which implicates that you are relinquishing some control. Life is full of choices.

**Comment by hacktivista** [comment]
`Life is full of choices`
xD agreed

One thing is use a package and other is changing how it's supposed to work, if `flymake-diagnostic-functions` is supposed to handle just 1 input, it would not be named with an `s`. I've commented back on the issue.

I'll stop arguing now.

**Comment by joaotavora** [comment]
> flymake-diagnostic-functions were supposed to handle just 1 input, it would not be named with an s

Did you stop to check the value of that variable anywhere in those two hours or were they all spent composing these silly comments?  You know like `C-h v`? Did it not say it is a list?  Did it not say that there's a global and a local value to it?

> I'll stop arguing now.

Good call.

**Comment by hacktivista** [comment]
No need to insult ;)

I'm not an emacs expert, I had to dig to check what was going on.

**Comment by joaotavora** [comment]
Those diagnostics are not quite on the level of Emacs expertise.  They are well documented and are basic user interfaces.

I didn't mean to insult you. I'm sorry for using the word "silly" but you are insisting on a point that I already explained.  I created Flymake with the multi-backend feature, and when I created Eglot I dialed it back for Eglot users actually using Eglot.  Simple as that. And I gave users the option to bring it back on at their own risk.

Now, if you think there's the need for documentation, I fully agree, _and that is why I recommended you open this PR_.  Now I think it is too wordy.  Would you be so kind as to reformulate it?  If you are, fine. If you aren't, go on with your life and I'll take 5 minutes later to do it myself.
